build_tools/docker/cmake-python-vulkan/Dockerfile

```diff
@@ -17,22 +17,15 @@
 
 FROM gcr.io/iree-oss/cmake-python AS final
 
-# It would be nice to have a separate install vulkan image and copy from that,
-# but I don't know all the files that installing the vulkan-sdk adds.
-RUN apt-get update && apt-get install -y wget
+ARG VULKAN_SDK_VERSION=1.2.154.0
```

Comment:
Now that we have a separate vulkan image, do these intermediate images actually provide value, or would it be better to just have cmake-python-nvidia install vulkan and nvidia and cmake-python-swiftshader install vulkan and swiftshader?

Reply:
I think they still provide value. They contain `VULKAN_SDK` and `PATH` env var; also symlink libvulkan to system location. Otherwise those will be duplicated in child images.